taotie/sources/arxiv.py

```diff
@@ -29,6 +32,9 @@ async def _cleanup(self):
         pass
 
     async def run(self):
+        with open('arxiv_author.json') as f:
+            author_dict = json.load(f)
+        self.authors = author_dict
```

Comment:
Please check the format of the data you load from the file. It is a dict, where each value is a list of authors. You may process them correctly.